grails-app/views/emails/_email_share_invite.gsp

```diff
@@ -1,7 +1,7 @@
 <p>Hello,<br></p>
 
 <p>${sharer} wants to share a canvas with you on the <a href="https://www.streamr.com/core/">Streamr Core</a> real-time data app.<br/>
```

Comment:
Hard-coded address should be replaced with `<g:link uri="/core" absolute="true"/>` or similar